query-engine/connectors/sql-query-connector/src/database/mysql.rs

```diff
@@ -44,22 +44,22 @@ impl FromSource for Mysql {
         features: psl::PreviewFeatures,
     ) -> connector_interface::Result<Mysql> {
         if source.provider == "@prisma/mysql" {
-            #[cfg(feature = "nodejs-drivers")]
+            #[cfg(feature = "js-drivers")]
             {
                 let driver = super::js::registered_driver();
                 let connection_info = get_connection_info(url)?;
 
                 return Ok(Mysql {
-                    pool: RuntimePool::NodeJS(driver.unwrap().clone()),
+                    pool: RuntimePool::Js(driver.unwrap().clone()),
                     connection_info,
                     features: features.to_owned(),
                 });
             }
 
-            #[cfg(not(feature = "nodejs-drivers"))]
+            #[cfg(not(feature = "js-drivers"))]
             {
                 return Err(ConnectorError::from_kind(ErrorKind::UnsupportedConnector(
-                    "The @prisma/mysql connector requires the `nodejs-drivers` feature to be enabled.".into(),
+                    "The @prisma/mysql connector requires the `js-drivers` feature to be enabled.".into(),
```

Comment:
Notice that here we're referring to the PSL preview feature name:

```suggestion
                    "The @prisma/mysql connector requires the `nodeDrivers` preview feature to be enabled.".into(),
```